<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Find Blocks</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Replace Blocks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="findReplaceTable">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Old Blocks</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Blocks</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="replaceBlocksInSelectionCheckbox">
           <property name="text">
            <string>In Selection</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="blocksReplaceButton">
           <property name="text">
            <string>Replace</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Find/Replace Items</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Find/Replace in Command Blocks</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Find/Replace NBT Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="tagNamesButton">
           <property name="text">
            <string>Tag Names</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="valueCheckbox">
           <property name="text">
            <string>Values</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="valueField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Tag Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="tagTypeComboBox">
           <item>
            <property name="text">
             <string>Any</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Any Numeric</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Any Integer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Any Floating-Point</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Any Array</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>String</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Byte</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Short</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Int</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Long</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Float</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Double</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>List</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Compound</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Byte Array</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Int Array</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Short Array</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Tag Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tagPathBox">
           <property name="placeholderText">
            <string>Path to search in, such as 'Data/Player/Inventory'</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tagPathHelpButton">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_24">
         <item>
          <widget class="QCheckBox" name="replaceNameCheckbox">
           <property name="text">
            <string>Replace Name With:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="replaceNameField"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="replaceValueCheckbox">
           <property name="text">
            <string>Replace Value With:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="replaceValueField"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tag Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="replaceTagTypeWithComboBox">
           <item>
            <property name="text">
             <string>Existing</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>String</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Byte</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Short</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Int</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Long</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Float</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Double</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QCheckBox" name="searchEntitiesCheckbox">
           <property name="text">
            <string>Search Entities</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="entityIDField">
           <property name="placeholderText">
            <string>Entity IDs, separted by ';'</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QCheckBox" name="searchItemsCheckbox">
           <property name="text">
            <string>Search Items</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="itemIDField">
           <property name="placeholderText">
            <string>Item IDs, separated by ';'</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QCheckBox" name="searchPlayersCheckbox">
           <property name="text">
            <string>Search Players</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="searchChunksCheckbox">
           <property name="text">
            <string>Search Chunks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="searchOtherDatsCheckbox">
           <property name="text">
            <string>Search other .dat files</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="nbtResultsView">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Progress:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="nbtProgressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nbtInSelectionCheckbox">
           <property name="text">
            <string>In Selection</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nbtReplaceSelectedButton">
           <property name="text">
            <string>Replace Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nbtReplaceAllButton">
           <property name="text">
            <string>Replace All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nbtFindButton">
           <property name="text">
            <string>Find</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
